### DISCUSSION TITLE
Something went wrong  NetworkError when attempting to fetch  - NetworkError when attempting to fetch resource.

### DISCUSSION THREAD

**Discussion by shuther** [marimo-team/marimo#2297] [General]
### Describe the bug

While it worked, I had to restart my computer and then, I was not able to connect to marimo again.
I tried the url: http://marino-vm.local:2718/ (so the main site)
Is there a place where I could see the logs on the server? 

### Environment

`marimo edit --headless --host 0.0.0.0 --port 2718 --token-password xxx`

### Code to reproduce

connection from the browser, error message:
```
Something went wrong

NetworkError when attempting to fetch resource.

If this is an issue with marimo, please report it on GitHub.
```

**Comment by mscolnick** [comment]
@shuther going to move this to a discussion

You can view the logs with `marimo -d edit ...` or with `--log-level`. You can run `marimo edit --help` to learn more.

Since it seem you are trying to run marimo behind a proxy, you may want o use `--proxy` (also described under the `help` command)

**Comment by shuther** [comment]
I can't reproduce it for now. I am not using a proxy but I will add more logs in the future
